### ISSUE TITLE
Write spec on command links

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical-web-and-design/vanilla-framework#3923]

**Comment by clagom** [comment]
@lyubomir-popov to add the Figma file to gather feedback from the team.

**Comment by lyubomir-popov** [comment]
Benchmarking https://www.figma.com/file/mA8jPxvPpkbKPxXLXZfqL7/Command-link-benchmarking?node-id=2%3A10

**Comment by amylily1011** [comment]
Command link spec: https://docs.google.com/document/d/1IPuq76HRCODYgxLkaMHbG4Ts9UU-ozK-E2FOiny4RO4/edit#heading=h.bqkuzi7416pz

**Comment by elioqoshi** [comment]
This looks good to me @amylily1011! Just a little comment on whether variant C can be defined as Anchor links

**Comment by cristinadresch** [comment]
@amylily1011 you're happy to put this in discourse now that has been reviewed?

**Comment by amylily1011** [comment]
I just need to make some correction and I'll put it on discourse.

**Comment by amylily1011** [comment]
@elioqoshi I think there will be just 2 variants - progressive disclosure and anchor link and I will merge variant B and C together. I think that makes more sense.

**Comment by amylily1011** [comment]
@lyubomir-popov here is the spec!

**Comment by lyubomir-popov** [comment]
Thanks @amylily1011, looks great. There are a couple of visual things to resolve but those will be on a separate Visual design issue.

**Comment by amylily1011** [comment]
Awesome @lyubomir-popov, give me a ping when you got those tickets done and I'll add the visuals to the doc and publish it 😄 
Should we close this ticket for now and create another one for publish command links? (@cristinadresch)

**Comment by amylily1011** [comment]
https://discourse.ubuntu.com/c/design/pattern-command-links/100/l/latest

**Comment by elioqoshi** [comment]
Thanks Amy! The Design specs need to be merged into one single post on Discourse however, could you do that and update here again? The Accessibility docs can be separate.

**Comment by clagom** [comment]
@amylily1011 What is the status of this? I think @marinacastejon pointed out an accessibility issue with the underline link, too?

**Comment by marinacastejon** [comment]
Hi, I left a comment in an[ issue](https://app.zenhub.com/workspaces/-design-system-tribe-59438c953747487777fbe484/issues/canonical-web-and-design/vanilla-framework/3350) that already existed, currently assigned to Lyubo.

**Comment by amylily1011** [comment]
@clagom, it's published on discourse but I need to consolidate this into 1 single page (as suggested by @elioqoshi ). @marinacastejon Created a separate issue for accessibility and attached it to the doc.

**Comment by clagom** [comment]
@amylily1011 @marinacastejon thanks we have this issue too: #3350 to discuss and define in this context

**Comment by clagom** [comment]
WG: this issue might be blocked by: #3350

**Comment by amylily1011** [comment]
This is not blocked anymore, @lyubomir-popov added new visuals to the spec and the spec is updated on the ubuntu blog https://discourse.ubuntu.com/t/about-the-pattern-command-links-category/26520
